Source/OxyPlot/Utilities/Helpers.cs

```diff
@@ -49,8 +49,8 @@ public static void Swap<T>(ref T value, ref T other)
                 throw new ArgumentNullException(nameof(projection));
             }
 
-            T best = default;
-            TComparable bestComparable = default;
+            T? best = default;
+            TComparable? bestComparable = default;
```

Comment:
For some reason I had to add these, otherwise it wouldn't compile. I don't know why this wasn't needed before...

Reply:
I'm still not sure on the rules with generics; I assume it's a C#9 change